pom.xml

```diff
@@ -37,6 +37,7 @@
   <properties>
     <jenkins.version>1.580.1</jenkins.version>
     <java.level>6</java.level>
+    <java.test.level>8</java.test.level>
```

Comment:
Do not use this property. If you are not updating the Jenkins baseline, test sources must remain at `-source 6`.